package.json

```diff
@@ -7,8 +7,11 @@
   "scripts": {
     "dev": "nodemon --inspect",
     "start": "probot run ./index.js",
-    "test": "npm-run-all --print-label --parallel lint:* --parallel test:*",
-    "lint:es": "eslint .",
+    "test": "npm-run-all --print-label --parallel test:*",
```

Comment:
Great work! With lint gone from `npm test` it would be awesome to add either a CONTRIBUTING.md file (or a simple "Development" heading in the repo readme) that states that it's expected to successfully run `npm lint` and `npm test` before posting a PR.